bundles/org.openhab.binding.shelly/src/main/java/org/openhab/binding/shelly/internal/ShellyBindingConstants.java

```diff
@@ -0,0 +1,257 @@
+    public static final String PROPERTY_UPDATE_NEW_VERS = "updateNewVersion";
+    public static final String PROPERTY_COAP_DESCR = "coapDeviceDescr";
+
+    // Relay
+    public static final String CHANNEL_GROUP_RELAY_CONTROL = "relay";
+    public static final String CHANNEL_OUTPUT = "output";
+    public static final String CHANNEL_INPUT = "input"; // prefix for input1, input2...
+    public static final String CHANNEL_INPUT1 = "input1";
+    public static final String CHANNEL__INPUT2 = "input2";
+    public static final String CHANNEL_OVERPOWER = "overpower";
+    public static final String CHANNEL_OVERTEMP = "overtemp";
+    public static final String CHANNEL_BRIGHTNESS = "brightness"; // Dimmer / Blub
+
+    public static final String CHANNEL_TIMER_AUTOON = "autoOn";
+    public static final String CHANNEL_TIMER_AUTOOFF = "autoOff";
+    public static final String CHANNEL_TIMER_ACTIVE = "timerActive";
+
+    // Roller
+    public static final String CHANNEL_GROUP_ROL_CONTROL = "roller";
+    public static final String CHANNEL_ROL_CONTROL_CONTROL = "control";
+    public static final String CHANNEL_ROL_CONTROL_POS = "rollerpos";
+    public static final String CHANNEL_ROL_CONTROL_TIMER = "timer";
+    public static final String CHANNEL_ROL_CONTROL_STOPR = "stopReason";
+    public static final String CHANNEL_ROL_CONTROL_DIR = "lastDirection";
+
+    // Dimmer
+    public static final String CHANNEL_GROUP_DIMMER_CONTROL = CHANNEL_GROUP_RELAY_CONTROL;
+
+    public static final String CHANNEL_GROUP_DIMMER_STATUS = "status";
+    public static final String CHANNEL_DIMMER_LOAD_ERROR = "loaderror";
+    public static final String CHANNEL_DIMMER_OVERLOAD = "overload";
+
+    // Power meter
+    public static final String CHANNEL_GROUP_METER = "meter";
+    public static final String CHANNEL_GROUP_METER1 = CHANNEL_GROUP_METER + "1";
+    public static final String CHANNEL_METER_CURRENTWATTS = "currentWatts";
+    public static final String CHANNEL_METER_LASTMIN1 = "lastPower1";
+    public static final String CHANNEL_METER_LASTMIN2 = "lastPower2";
+    public static final String CHANNEL_METER_LASTMIN3 = "lastPower3";
+    public static final String CHANNEL_METER_TOTALKWH = "totalKWH";
+    public static final String CHANNEL_EMETER_TOTALRET = "returnedKWH";
+    public static final String CHANNEL_EMETER_REACTWATTS = "reactWatts";
+    public static final String CHANNEL_EMETER_VOLTAGE = "voltage";
+    public static final String CHANNEL_METER_TIMESTAMP = "timestamp";
+
+    public static final String CHANNEL_GROUP_LED_CONTROL = "led";
+    public static final String CHANNEL_LED_STATUS_DISABLE = "statusLed";
+    public static final String CHANNEL_LED_POWER_DISABLE = "powerLed";
+
+    public static final String CHANNEL_GROUP_SENSOR = "sensors";
+    public static final String CHANNEL_SENSOR_TEMP = "temperature";
+    public static final String CHANNEL_SENSOR_TUNIT = "tempUnit";
```

Comment:
This is a device config option, not a channel. Please remove.

Reply:
Why should we require the user to set this config in the Shelly App AND the binding config. 
tempUnit is read from the device status and can be used by ruiles etc. to find out of the temperature value is in C or F. So if the user changes the device setting in the Shelly App or Web Ui the binding automatically reports this value without another config change.

Reply:
You got me wrong, thic can be read from the device and be stored as a device config for further usage, but there is no need to have a channel which displays this setting.

Reply:
Do you mean Thing Config? How to access device/thing config from a rule?

Reply:
channel removed, binding converts from F to C, channel gets updated with quantity type CELSIUS. The framework should convert when system config is set to F.
ok?